upstream-community-operators/azure-service-operator/0.37.0/azure-service-operator.v0.37.0.clusterserviceversion.yaml

```diff
@@ -1193,6 +1195,8 @@ spec:
 
     4. Next, we'll create a Service Principal with Contributor permissions for your subscription, so ASO can create resources in your subscription on your behalf. Note that the [ServicePrincipal](https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli) you pass to the command below needs to have access to create resources in your subscription. If you'd like to use Managed Identity for authorization instead, check out instructions [here](https://github.com/Azure/azure-service-operator/blob/master/docs/howto/managedidentity.md).
 
+        Note: Please make sure that the Service Principal name mentioned is unique otherwise you will see `Found an existing application instance` warning message if you run below instuction.
```

Comment:
Next, we'll create a Service Principal with Contributor permissions for your subscription, so ASO can create resources in your subscription on your behalf. Note that the [ServicePrincipal](https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli) you pass to the command below needs to have access to create resources in your subscription. If you'd like to use Managed Identity for authorization instead, check out instructions [here](https://github.com/Azure/azure-service-operator/blob/master/docs/howto/managedidentity.md).

If creating a Service Principal ensure that the name used for it is unique in your AAD tenant. If it is not, you may see an error `Found an existing application instance`

        ```sh
        az ad sp create-for-rbac -n "<your-service-principal-name>" --role contributor \
            --scopes /subscriptions/$AZURE_SUBSCRIPTION_ID
        ```

        This should give you output like the following:
        ```sh
        "appId": "xxxxxxxxxx",
        "displayName": "<your-service-principal-name>",
        "name": "http://<your-service-principal-name>",
        "password": "xxxxxxxxxxx",
        "tenant": "xxxxxxxxxxxxx"
        ```

Reply:
I suggest this because people are liable to copy-paste things and the hard-coded SP name is likely to fail them.